test/dynamo/test_modules.py

```diff
@@ -1993,6 +2074,55 @@ def fn(x):
         self.assertEqual(ref, res)
 
 
+import torch.distributed as dist
```

Comment:
@voznesenskym This should be a drop-in replacement from L2077 to L2123

https://gist.github.com/wanchaol/c40bf988fa8a822d93527dcbbb7eae3e